### DISCUSSION TITLE
Adding Excalidraw libraries / folder location?

### DISCUSSION THREAD

**Discussion by NedPool** [TriliumNext/Trilium#3663] [Q&A]
I absolutely love Trilium and the ability to self-host important data like this.

After playing with the canvas notes a bit, I would like to add some of the existing Excalidraw shape libraries to my Trilium install, but I can't seem to get it working. After clicking the "browse libraries" button in the canvas note, the Excalidraw links are coming up with my domain and path for Trilium and the #addLibrary arg seems correct on the URL, but clicking that "Add to Excalidraw" link for the library just opens a new tab in my Trilium. I can download the libraries and don't have an issue with uploading them manually to Trilium, but if that folder exists, it is not covered in any install documentation and I can't find it roaming through my docker instance.

Am I missing/can't find the folder for these libraries or is there an issue with adding them in the current iteration of Trilium?

**Comment by zadam** [comment]
Hello, you're not missing anything. The integration with excalidraw is basic so far and the libraries are not supported (yet).

**Reply by NedPool** [reply]
I completely understand and thought that might be the case.

I look forward to future development and thank you for your effort on this project!
